### PR TITLE
feat: add section 295 and 302

### PR DESCRIPTION
**Pull request by tamer545**
### What?
Added section 295 and 302
### Why?
Because these sections are needed for the book
### How?
By creating a yaml definition
### Testing?
Tested for syntax errors
### Screenshots (for frontend changes)
No
### Anything Else?
#85 
#219

### CODE REVIEW

File: fabbwled-backend/src/main/resources/sections/289-324.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+- number: 295
+  content:
+    - text: The  scorpion  men  cannot  keep  up  with  you,  and  you  leg  it  all the  way  back  to  the  road.  Maybe  you  can  come  back  and  try again later.
```
Comment (Nilstrieb) [inline fabbwled-backend/src/main/resources/sections/289-324.yaml:3]: the spaces here are broken, can you fix them

**Review by Nilstrieb — CHANGES_REQUESTED** [review]

**Review by Nilstrieb — APPROVED** [review]

**Review by adrsch04 — APPROVED** [review]
It looks good; maybe add newlines between the sections to improve readability.

**Review by Desi7 — APPROVED** [review]
Looks good to me 👍

**Review by dbabvwid — APPROVED** [review]

**Review by kimberlymo — APPROVED** [review]
lgtm 👍

**Pull request merged**